driver-sync/src/test/functional/com/mongodb/client/JsonPoweredCrudTestHelper.java

```diff
@@ -126,9 +125,6 @@ BsonDocument getOperationResults(final BsonDocument operation, @Nullable final C
         } catch (NoSuchMethodException e) {
             throw new UnsupportedOperationException("No handler for operation " + methodName);
         } catch (InvocationTargetException e) {
-            if (e.getTargetException() instanceof AssumptionViolatedException) {
```

Comment:
This is no longer necessary, and removing it means that the executor doesn't need to take a Junit dependency.

Reply:
Ack